src/components/YourCompany/YourCompany.vue

```diff
@@ -421,15 +424,31 @@ export default class YourCompany extends Mixins(CommonMixin, DateMixin, LegalApi
 }
 
 .company-name {
-  font-size: 1.375rem;
-  font-weight: bold
+  font-size: 1.5rem;
 }
 
 .company-info {
+  font-size: 1rem;
+  padding-top: 0.5rem;
+  color: $gray7;
+}
+
+.subtitle {
+  color: $gray9;
+  font-weight: bold;
+}
+
+.hasConflict {
+  color: $BCgovError;
+}
+
+.name-request-applicant-info {
+  font-size: 1rem;
+  color: $gray7;
   padding-top: 0.5rem
 }
 
-.name-request-applicant-info:not(:first-child) {
+.name-request-applicant-info :not(:first-child) {
```

Comment:
Are you sure about this change?

This changes from "not the first child name-request-applicant-info" to "not the first GRAND-child of name-request-applicant-info". (I think.)

Reply:
Sorry, this could be a typo. Will check

Reply:
WAS a typo, thanks for this!